buildchain/buildchain/image.py

```diff
@@ -280,6 +280,19 @@ def _local_image(name: str, **kwargs: Any) -> targets.LocalImage:
             "ETCD_VERSION": f"v{versions.ETCD_VERSION}",
         },
     ),
+    _local_image(
+        name="metalk8s-operator",
+        dockerfile=constants.METALK8S_OPERATOR_ROOT / "Dockerfile",
+        build_context=constants.METALK8S_OPERATOR_ROOT,
+        build_args={
+            "BUILD_DATE": datetime.datetime.now(datetime.timezone.utc)
+            .astimezone()
+            .isoformat(),
+            "VCS_REF": constants.GIT_REF or "<unknown>",
+            "METALK8S_VERSION": versions.VERSION,
+            "VERSION": "latest",
```

Comment:
Why are we doing this? Couldn't we just tag our operators with the project version? (I see we actually use the version in the image tag, but this label should reflect the same value I think)

Reply:
TBH I don't really know, I did the same as for the storage operator, I think you should ask it to @gdemonet who did it for storage operator https://github.com/scality/metalk8s/commit/fdeae9674719c683e4955de7f081da6087fc32bc :stuck_out_tongue_winking_eye:

Reply:
:trollface:

Reply:
OK, let's not change storage-operator right now, but can you change this line to:

```suggestion
            "VERSION": versions.VERSION,
```

please?

Reply:
Sure I will do it

Reply:
Done